docs/changes.rst

```diff
@@ -50,6 +50,35 @@ The best check that everything is ok is to run a dry-run extraction::
     borg extract -v --dry-run REPO::ARCHIVE
 
 
+Version 1.1.0b2 (not released yet)
+----------------------------------
+
+Bug fixes:
+
+- fix corruption caused by incorrect preservation of delete tags, leading to
+  "object count mismatch" on borg check, #1598
```

Comment:
+ This only occurred with 1.1.0b1 and is normally fixed by running another borg create/delete/prune.